### PR TITLE
Fixed Player object in Joined event being null

### PR DESCRIPTION
**Pull request by Misaka-ZeroTwo**

### CODE REVIEW

**Comment by Misaka-ZeroTwo** [comment]
Player is no longer null, yet Player.UserId and Player.Nickname is still null. But it is called even after NW API join event. How could it be null when it's not in NW API???

File: Exiled.Events/Patches/Events/Player/Joined.cs [hunk 1]
```diff
@@ -59,46 +59,18 @@ internal static void CallEvent(ReferenceHub hub, out Player player)
...
-                    new(OpCodes.Call, Method(typeof(Joined), nameof(CallEvent))),
-                });
-
-            for (int z = 0; z < newInstructions.Count; z++)
-                yield return newInstructions[z];
-
-            ListPool<CodeInstruction>.Pool.Return(newInstructions);
+            if (ReferenceHub.AllHubs.Count - 1 < CustomNetworkManager.slots)
+            {
+                CallEvent(__instance, out _);
```
Comment (louis1706) [inline Exiled.Events/Patches/Events/Player/Joined.cs:68]: this should always being call

**Review by louis1706 — CHANGES_REQUESTED** [review]

File: Exiled.Events/Patches/Events/Player/Joined.cs (hunk 1 above)
Comment (louis1706) [reply Exiled.Events/Patches/Events/Player/Joined.cs:68]: the code does that in IL 
```
            if (ReferenceHub.AllHubs.Count - 1 < CustomNetworkManager.slots)
                            MultiAdminFeatures.CallEvent(MultiAdminFeatures.EventType.SERVER_FULL);
            CallEvent(__instance, out _);
          ```

Comment (louis1706) [reply Exiled.Events/Patches/Events/Player/Joined.cs:68]: the code does that in IL 
```
            if (ReferenceHub.AllHubs.Count - 1 < CustomNetworkManager.slots)
                            MultiAdminFeatures.CallEvent(MultiAdminFeatures.EventType.SERVER_FULL);
            CallEvent(__instance, out _);
          ```

**Review by louis1706 — APPROVED** [review]

**Review by louis1706 — CHANGES_REQUESTED** [review]
Fix error

**Review by louis1706 — APPROVED** [review]

**Comment by Misaka-ZeroTwo** [comment]
Macos + github.dev moment

**Comment by Michal78900** [comment]
Update branch

**Comment by Misaka-ZeroTwo** [comment]
done

**Review by Michal78900 — APPROVED** [review]

**Pull request merged**